samples/bluetooth/central_smp_client/README.rst

```diff
@@ -1,13 +1,13 @@
 .. _bluetooth_central_dfu_smp:
 
-Bluetooth: Central DFU SMP
-##########################
+Bluetooth: Central SMP Client
+#############################
 
 .. contents::
    :local:
    :depth: 2
 
-The Central DFU SMP sample demonstrates how to use the :ref:`dfu_smp_readme` to connect to an SMP Server and send a simple echo command.
+The Central SMP Client sample demonstrates how to use the :ref:`dfu_smp_readme` to connect to an SMP Server and send a simple echo command.
 The response, which is received as CBOR-encoded data, is decoded and printed.
 
 
```

Comment:
Below: "and configures the DFU SMP Client" should be updated as well.

Reply:
Oops, I missed that.

Reply:
You are right, my mistake.

BTW what is the reasoning for removing "dfu" from the sample name then? Seems to be less clear now.

Reply:
I think the "dfu" in the name may be confusing because the sample does not do Device Firmware Updates.Only uses DFU SMP module.

Reply:
Then by that same logic isn't the DFU in the "DFU SMP" module also confusing?

It makes sense to call it the central DFU_SMP client if it is using that module, now this looks incomplete to me.
@grochu Don't you think so as well?

Reply:
The thing is the DFU SMP Client can be used to transfer firmware image for update and trigger the update, and originally it was supposed to be used for this purpose in the nRF Desktop application for a USB dongle. However, in the end it was decided to use a different service for the DFU over BLE. So the term "_DFU_" for the Client library looks OK to me, while in the sample it was confusing, because clearly the sample only exchanges meaningless echo messages.

Reply:
Yeah, alright. I still think it looks strange since the sample is demonstrating the DFU_SMP module, but if that is only me I'll budge.